seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf

```diff
@@ -36,7 +36,7 @@ source {
     schema = {
       fields {
         c_map = "map<string, string>"
```

Comment:
Is there nothing wrong with MAP? Don't it need to convert to MapData?

Reply:
yes, it is. I will fix in another PR，not enough time for one day to fix it all. Besides and same to Date type

Reply:
OK, thanks for your feedback.